include/fastdds/rtps/attributes/ServerAttributes.h

```diff
@@ -124,6 +124,21 @@ RTPS_DllAPI bool get_server_client_default_guidPrefix(
 
 } // namespace rtps
 } // namespace fastdds
+
```

Comment:
`aux` namespace shows up in the doxygen documentation of `RemoteServerAttributes`
```
//!Metatraffic Unicast Locator List
aux::LocatorList_t metatrafficUnicastLocatorList;
//!Metatraffic Multicast Locator List.
aux::LocatorList_t metatrafficMulticastLocatorList;

//!Guid prefix
aux::GuidPrefix_t guidPrefix;
```

I think it's better to remove it all together

Reply:
👍